eyelid-match-ops/src/primitives/poly.rs

```diff
@@ -32,113 +28,3 @@ pub mod test;
 
 // Do not add code here.
 // Add functions or trait impls to modular_poly/*.rs and inherent method impls to modular_poly.rs.
-
-/// Implementation based on Algorithm 3.3.1 (Page 118) from
```

Comment:
Review note: I verified this code move using `git diff --color-moved main origin/ntru-keygen`